docs/concepts/what-is-a-workflow-id-reuse-policy.md

```diff
@@ -2,27 +2,22 @@
 id: what-is-a-workflow-id-reuse-policy
 title: What is a Workflow Id Reuse Policy?
 sidebar_label: Workflow Id Reuse Policy
-description: A Workflow Id Reuse Policy determines whether a Workflow Execution is allowed to spawn with a particular Workflow Id, if that Workflow Id has been used with a previous, and now Closed, Workflow Execution.
+description: Explanation of the three different Workflow Id Reuse Policies
```

Comment:
This should always be the same as the first sentence in the article.

We only do this for the guides that are generated from these source files.